armadillo/armadillo.py

```diff
@@ -94,6 +191,15 @@ def find_pi_stacking_rings(list_of_rings, distmax, distmin,
     --------
     Returns a lists with n = [0] being ix_of_actually_pi_stacking and n= [1] ix_of_none_pi_stacking
     '''
+
+    ring_cogs = []
+    for ring in ring_list:
+        ring_cogs.append(ring.center_of_geometry())
```

Comment:
why don't you use`ring_center`?
you could try list comprehension instead of a loop:)